cli/jobs/pipelines-with-components/shakespear_sample_and_word_count_using_spark/README.md

```diff
@@ -1,17 +0,0 @@
----
-page_type: sample
```

Comment:
confirm: is this a by-design delete?

Reply:
It is by accident, I already bring back spark component exmaple.